packages/core/src/__tests__/ceramic-anchor.test.ts

```diff
@@ -319,33 +319,6 @@ describe('Ceramic anchoring', () => {
     await ceramic2.close()
   })
 
-  it('test the same doc anchored twice (same Ceramic instance), first one wins', async () => {
```

Comment:
You were totally right about this test Janko, my mistake.  Because they share a ceramic instance they learn about each other's updates instantly and there's no real conflict resolution behavior being tested.  Sorry I didn't follow what you were getting at on the other review!

Reply:
nice, cool!